code/__HELPERS/text.dm

```diff
@@ -456,32 +456,32 @@ proc/checkhtml(var/t)
 
 
 // Pencode
-/proc/pencode_to_html(text, mob/user, obj/item/weapon/pen/P = null, sign = 1, fields = 1, deffont = PEN_FONT, signfont = SIGNFONT, crayonfont = CRAYON_FONT)
-	text = replacetext(text, "\n",			"<BR>")
-	text = replacetext(text, "\[center\]",	"<center>")
-	text = replacetext(text, "\[/center\]",	"</center>")
-	text = replacetext(text, "\[br\]",		"<BR>")
+/proc/pencode_to_html(text, mob/user, obj/item/weapon/pen/P = null, format = 1, sign = 1, fields = 1, deffont = PEN_FONT, signfont = SIGNFONT, crayonfont = CRAYON_FONT)

```

Comment:
Make sure to update every other call of `pencode_to_html` to expect the `format = 1` field.